### ISSUE TITLE
Nested expressions

### ISSUE THREAD

**Issue by jeandrek** [AceLewis/my_first_calculator.py#24]
Like
`(2 * 6) + 9`

We really need this

**Comment by rudrathegreat** [comment]
Another way to do this is to simply find `2*6` and then add `9` to that. 

The program should have a functionality where it could remember the previous answer. But, maybe you could just write it down... on a book

**Comment by jeandrek** [comment]
You mean to say, I find 2\*6 and write a book with the answer as its title?

**Comment by rudrathegreat** [comment]
Not exactly. You basically find what `2*6` is and then write the answer for that expression in some notebook.

With the functionality of allowing the program to remember the previous answer, we could do something like this - 

https://gist.github.com/rudrathegreat/9fa8578ae83664baef182948d4f73dac
